Arrangement_on_surface_2/include/CGAL/Arrangement_on_surface_2.h

```diff
@@ -1516,6 +1527,37 @@ class Arrangement_on_surface_2 {
 
   //@}
 
+  /*!
+   * Cleans the inner CCB if sweep mode was used, by removing all
+   * non-valid inner CCBs
+   */
+  void clean_inner_ccbs()
```

Comment:
the function name should remind that is it is related to a post sweep

Reply:
Indeed.